internal/controller/infrastructure/kinkmachine_controller.go

```
@@ -19,12 +19,19 @@
 
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/log"
 )
 
+const (
+	// name of control-plane finalizer
+	machineFinalizer = "machine.kink.anza-labs.com/finalizer"
```

golangci-lint (1.62.2)

const `machineFinalizer` is unused

(unused)